src/public/modules/forms/helpers/__tests__/FeedbackCsvGenerator.spec.ts

```diff
@@ -65,5 +65,33 @@ describe('FeedbackCsvGenerator', () => {
         expect.stringContaining(todaysDate),
       )
     })
+
+    it('should add single quote before comment if comment starts with a formula character', () => {
+      // Arrange
+      const feedbackCsv = new FeedbackCsvGenerator(1)
+      const feedback: FormFeedbackDto = {
+        rating: 3,
+        comment: '=formula',
+        formId: new ObjectId().toHexString(),
+        created: new Date('2019-11-05T13:12:14').toISOString() as DateString,
+        lastModified: new Date(
+          '2019-11-05T13:12:14',
+        ).toISOString() as DateString,
+      }
+      const insertedCreatedDate = moment(feedback.created)
+        .tz('Asia/Singapore')
+        .format('DD MMM YYYY hh:mm:ss A')
+
+      const insertedLine = `${insertedCreatedDate},'${feedback.comment},${feedback.rating}`
```

Comment:
@wanlingt there's a bug here right? there should be a single quote in the expected line

Reply:
or actually should not have anything here since there was no change to AngularJS's feedback csv generator.

```suggestion
      const insertedLine = `${insertedCreatedDate},${feedback.comment},${feedback.rating}`
```

Reply:
my bad, you are right! this tests for feedback csv generator in AngularJS not React. Will have to add a separate test for React

Reply:
fixed in your PR [8b33acc](https://github.com/opengovsg/FormSG/pull/5010/commits/8b33acc7ab4d3896b83b18548eb0982660a2e0a3)